src/genTempId.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Generates a temporary id which is semi-unique.
+ */
+export function genTempId(): string {
+  return String(Math.floor((1 + Math.random()) * 0x10000));
```

Comment:
my pr's bringing in a uuid lib capabile of generating uuidv4 guids if you want to use it.